### PR TITLE
Template for PRs - to automate 

### PR DESCRIPTION
**Pull request by zees-dev**
# Changelog

A template for PRs - with a dedicated section for core changes (which could later be used for release note generation)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,5 +1,120 @@
...
+
+- [ ] Unit tests added/updated
+- [ ] Integration tests added/updated
+- [ ] Manual testing performed
+
+---
+
+<!-- This section will be used to automatically generate release notes -->
+<!-- Please fill out relevant sections below based on your changes -->
+<!-- Leave sections empty if there are no changes -->
```
Comment (JasonTulp) [inline .github/pull_request_template.md:27]: Personally I think we should delete sections that aren't being used. For example EVM Precompile changes
It should be easy enough to automate it even with missing parts

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,5 +1,120 @@
...
+
+## Related Issues
+
+- _Link any related issues_
+
+## Testing
+
+- [ ] Unit tests added/updated
+- [ ] Integration tests added/updated
+- [ ] Manual testing performed
```
Comment (JasonTulp) [inline .github/pull_request_template.md:21]: Not sure if this part is necessary

File: .github/pull_request_template.md (hunk 1 above)
Comment (zees-dev) [reply .github/pull_request_template.md]: addressed: https://github.com/futureversecom/trn-seed/pull/899/commits/8150746fd053fe0ef3b34fe3a15982468b634c2f

File: .github/pull_request_template.md (hunk 2 above)
Comment (zees-dev) [reply .github/pull_request_template.md]: addressed: https://github.com/futureversecom/trn-seed/pull/899/commits/3a0878ab19f5a2c2bd285a922230761aad07c1d0

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,5 +1,114 @@
...
+
+- [ ] Runtime Changes
+- [ ] Client Changes
+
+## API Changes
+
+### EVM Precompile Changes
+
+#### Added
+
```
Comment (surangap) [inline .github/pull_request_template.md:37]: Maybe add a comment on how we want to add the changes here. i.e `<pallet runtime name>: <change>`

**Review by surangap — APPROVED** [review]
lgtm

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,5 +1,114 @@
...
+-
+
+#### Removed
+
+-
+
+### Storage Changes
+
+#### Added
+
```
Comment (surangap) [inline .github/pull_request_template.md:51]: Maybe add a comment on how we want to add the changes here. i.e <pallet runtime name>: <change>

Comment (zees-dev) [reply .github/pull_request_template.md:51]: addressed: https://github.com/futureversecom/trn-seed/pull/899/commits/f98b04554d4267cf9c225d7e6f359042918b596a

**Review by JasonTulp — APPROVED** [review]
lgtm!

**Pull request merged**